DDDEastAnglia.Tests/DateExtensions.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+namespace DDDEastAnglia.Tests
+{
+    public static class DateExtensions
+    {
+        public static DateTime January(this int day, int year)
```

Comment:
Noice :smile: :+1: 

(Sorry for the triviality, but please could you reformat - blank lines between methods, no blank line after last method?)